iXBRLViewerPlugin/__init__.py

```diff
@@ -40,6 +40,9 @@
 from .localviewer import launchLocalViewer, VIEWER_BASENAME_SUFFIX
 from arelle.ModelDocument import Type
 
+import traceback
+import sys
```

Comment:
Remove unused?

Reply:
They are used (see [line 74](https://github.com/Workiva/ixbrl-viewer/pull/228/files/1751ec0fcef4eabed6e1d81bc0a25a40fb3e3cb2#diff-a65816b2086d1b8e75aabec4dd3761c8795d7693a1d186400b4718c1a894fceaR74))

I managed to trigger an exception during development which highlighted that these imports were missing.

Reply:
Ah, thanks, I overlooked the commit message, sorry.